_toc.yml

```diff
@@ -59,6 +60,7 @@ entries:
         title: HowTo
         entries:
           - file: docs/products/kafka/howto/kafka-sasl-auth
+          - file: docs/products/kafka/howto/integrate-service-logs-into-kafka-topic
```

Comment:
I'm not sure what this change is doing here!

Reply:
😨 This was not supposed to be there! Thx for catching it. Removed